sqlmesh/dbt/basemodel.py

```diff
@@ -140,8 +142,20 @@ def _validate_columns(cls, v: t.Any) -> t.Dict[str, ColumnConfig]:
 
     @property
     def all_sql(self) -> SqlStr:
+        return SqlStr("\n".join([self.hook_sql, self.sql_no_config]))
```

Comment:
Does the order matter at all? Is it ok that the post hook preceedes sql?

Reply:
No. We are only capturing dependencies.